### PR TITLE
[JSON-RPC] Add `Addons` announcements

### PR DESCRIPTION
**Pull request by dmyoung9**
## Description

This allows for addons which have optional dependencies to listen during the dependency install process, and react accordingly.

## How has this been tested?

This is currently not working... My builds are successful, but I feel like I'm missing something, because the notifications are never actually sent, and hoping the team here may be able to help me smooth out the gaps.

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [x] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by lrusak** [comment]
Please make PRs to the master branch first.

**Comment by CastagnaIT** [comment]
commit titles dont follow our [contributing guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)
must be formatted as `[component/context] the appropriate title`

File: xbmc/interfaces/json-rpc/schema/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-JSONRPC_VERSION 13.0.0
+JSONRPC_VERSION 13.0.1
```
[the remark this answers is not in the document]
Comment (dmyoung9) [reply xbmc/interfaces/json-rpc/schema/version.txt]: Fixed in c8cb9f0 :+1:

**Comment by dmyoung9** [comment]
> commit titles dont follow our [contributing guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md) must be formatted as `[component] the appropriate title`

Fixed :+1:

**Comment by CastagnaIT** [comment]
> This is currently not working... 

WIP pr's should be marked as draft
if you can add a little example on how should be implemented in an addon

**Pull request closed without merging**